content/applications/marketing/events.rst

```diff
@@ -0,0 +1,16 @@
+:nosearch:
+:show-content:
+:hide-page-toc:
+:show-toc:
+
+======
+Events
+======
+
+.. seealso::
+   - `Odoo Tutorials: Events <https://www.odoo.com/slides/surveys-63>`_
+
+.. toctree::
+   :titlesonly:
+
+   events/overview
```

Comment:
Isn't that a bit strange to have a single sub-directory named "Overview"? Couldn't "Events" have "Event essentials", "Sell tickets", and "Track and manage talks" as direct sub-pages?

Reply:
Yes, I agree 👍